src/i18n/Localizable.xcstrings

```diff
@@ -19899,6 +20003,7 @@
       }
     },
     "Resume" : {
+      "comment" : "is this one used as verb or noun?",
```

Comment:
Verb

Reply:
changed noun to gerund. same reason for"Restart", `because in Korean noun or gerund is usually used to implement actions`